environments/softlayer/public.yml

```diff
@@ -124,10 +124,10 @@ KSPLICE_ACTIVE: yes
 AMQP_HOST: "{{ groups.rabbitmq.0 }}"
 AMQP_NAME: commcarehq
 
-kafka_broker_id: 0
-kafka_log_dir: '/opt/data/kafka'
+encrypted_root: "{{ encrypted_root_override|default('/opt/data/ecrypt') }}"
```

Comment:
Is that really the recommended default? I thought we had annoying issues with envs where `/opt/data/ecrypt` was used instead of `/opt/data`.

Reply:
@millerdev this is for the softlayer env, where all but a few machines use `/opt/data/ecrypt`. Now understanding this PR a bit better though, I have some thoughts about how I'd do this differently to highlight that `/opt/data` is really the preferred one—even though in this particular env its also the less common one.